runtimes/pythonrt/pythonrt.go

```
@@ -593,7 +566,7 @@ func (py *pySvc) drainPrints(ctx context.Context) {
 		case r := <-py.channels.log:
 			py.log.Log(pyLevelToZap(r.level), r.message)
 		case r := <-py.channels.print:
-			py.cbs.Print(ctx, py.runID, r.message)
+			_ = py.cbs.Print(ctx, py.runID, r.message)
```

Don't ignore the error